modules/nodes-scheduler-node-selectors-pod.adoc

```diff
@@ -60,11 +58,61 @@ The web console lists the controlling object under `ownerReferences` in the pod
 
 .Procedure
 
-. Add the desired label to your nodes:
+. Add labels to a node using a MachineSet or editing the node directly:
+
+* Use a MachineSet to add labels to nodes managed by the MachineSet when a node is created:
+
+.. Run the following command to add a node selector to a MachineSet:
++
+[source,terminal]
+----
+$ oc patch MachineSet <name> --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"<key>"="<value>","<key>"="<value>"}}]'  -n openshift-machine-api
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc patch MachineSet abc612-msrtw-worker-us-east-1c  --type='json' -p='[{"op":"add","path":"/spec/template/spec/metadata/labels", "value":{"type":"user-node","region":"east"}}]'  -n openshift-machine-api
+----
+
+.. Verify that the label is added to the MachineSet, use the `oc edit` command:
++
+For example:
++
+[source,terminal]
+----
+$ oc edit MachineSet ci-ln-l8nry52-f76d1-hl7m7-worker-c -n openshift-machine-api
```

Comment:
Where did we get the MachineSet? That's not the one you patched in the previous step.